src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -2033,7 +2025,7 @@ public void onBendingPlayerCreation(final BendingPlayerCreationEvent event) {
 		final Player player = event.getBendingPlayer().getPlayer();
 		BendingBoardManager.canUseScoreboard(player);
 	}
-
+	

```

Comment:
Accidental tab :)

Reply:
Gonna just take care of this because it is still there.